index.js

```diff
@@ -31,7 +31,7 @@ function test(p, opts) {
     ) {
       throw new Error(
         chalk.red('Code did not pass lint rules') +
-        formatter(report.results)
+        formatter(report.results).replace(process.cwd() + '/', '')
```

Comment:
Actually, do you think you could add a small comment to explain why the replace is needed?  I can see this causing some head-scratching in the future.